src/shared/components/Checkout/Checkout.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import CircularProgressbar from '../CircularProgressbar'
+import Text from '../Text'
+import Icon from '../Icon'
+import {
+  Container,
+  TopRowContainer,
+  StepsProgressContainer,
+  ProgressbarContainer,
+  StepsContainer,
+  Step,
+  StepInnerContainer,
+  StepState,
+  CheckIcon,
+  ChecvronIcon,
+} from './Checkout.styles'
+
+export type Step = { title: string; onClick: () => void; completed: boolean }
+export type CheckoutProps = {
+  steps: Step[]
+}
+export const Checkout: React.FC<CheckoutProps> = ({ steps }) => {
+  const stepsCompletedNumber = steps.filter(({ completed }) => completed).length
+  return (
+    <Container>
+      <TopRowContainer>
+        <Text variant="h6">Progress</Text>
+        <StepsProgressContainer>
+          <ProgressbarContainer>
+            <CircularProgressbar value={stepsCompletedNumber} maxValue={steps.length}></CircularProgressbar>
+          </ProgressbarContainer>
+          <Text variant="body2">
```

Comment:
Just noticed this but the progress circle actually moves based on the progress:

https://user-images.githubusercontent.com/12646744/111629868-07f0f780-87f2-11eb-8434-9562e03a0d5b.mov

You can fix this by attaching
```css
min-width: 25px;
text-align: right;
```

to this text

Reply:
Right, added the styles doesn't seem to move anymore